docs/api/instruments/anritsu/anritsuMS9740A.rst

```diff
@@ -0,0 +1,3 @@
+.. autoclass:: pymeasure.instruments.anritsu.AnritsuMS9740A
```

Comment:
```suggestion
#########################################
Anritsu MS9740A Optical Spectrum Analyzer
#########################################

.. autoclass:: pymeasure.instruments.anritsu.AnritsuMS9740A
```

you'll need this, too to make the warning below go away. 
I'll try to use these suggestions as much as possible, you basically only have to press an accept button somewhere here to use what I suggest.